iml-gui/crate/src/generated/css_classes.rs

```diff
@@ -3904,6 +3906,15 @@ pub struct CssClasses<'a> {
     pub hidden: &'a str,
 
     /**
+        display: block;
```

Comment:
Some sort of bug in the codgen? A lot of these look repeated.

Reply:
I think so.

Reply:
Is there anything we can do on our end to not have these added?

Reply:
Only to have a local modified copy of the generator? => https://github.com/MartinKavik/postcss-typed-css-classes/blob/3ad711589d475f07daa28b1581d8aaade394f71a/generators/rust_generator.js#L129

Reply:
Can you open an issue on their repo about it?

Reply:
https://github.com/MartinKavik/postcss-typed-css-classes/issues/3